drivers/SmartThings/zigbee-window-treatment/src/aqara/roller-shade/init.lua

```diff
@@ -0,0 +1,140 @@
+local capabilities = require "st.capabilities"
+local clusters = require "st.zigbee.zcl.clusters"
+local cluster_base = require "st.zigbee.cluster_base"
+local FrameCtrl = require "st.zigbee.zcl.frame_ctrl"
+local data_types = require "st.zigbee.data_types"
+local aqara_utils = require "aqara/aqara_utils"
+
+local Basic = clusters.Basic
+local WindowCovering = clusters.WindowCovering
+
+local initializedStateWithGuide = capabilities["stse.initializedStateWithGuide"]
+local reverseRollerShadeDir = capabilities["stse.reverseRollerShadeDir"]
+local shadeRotateState = capabilities["stse.shadeRotateState"]
+local setRotateStateCommandName = "setRotateState"
+
+local MULTISTATE_CLUSTER_ID = 0x0013
+local MULTISTATE_ATTRIBUTE_ID = 0x0055
+local ROTATE_UP_VALUE = 0x0004
+local ROTATE_DOWN_VALUE = 0x0005
+
+
+local function window_shade_level_cmd(driver, device, command)
+  -- Cannot be controlled if not initialized
+  local initialized = device:get_latest_state("main", initializedStateWithGuide.ID,
+    initializedStateWithGuide.initializedStateWithGuide.NAME) or 0
+  if initialized == initializedStateWithGuide.initializedStateWithGuide.initialized.NAME then
+    aqara_utils.shade_level_cmd(driver, device, command)
+  end
+end
+
+local function window_shade_open_cmd(driver, device, command)
+  -- Cannot be controlled if not initialized
+  local initialized = device:get_latest_state("main", initializedStateWithGuide.ID,
+    initializedStateWithGuide.initializedStateWithGuide.NAME) or 0
+  if initialized == initializedStateWithGuide.initializedStateWithGuide.initialized.NAME then
+    device:send_to_component(command.component, WindowCovering.server.commands.GoToLiftPercentage(device, 100))
+  end
+end
+
+local function window_shade_close_cmd(driver, device, command)
+  -- Cannot be controlled if not initialized
+  local initialized = device:get_latest_state("main", initializedStateWithGuide.ID,
+    initializedStateWithGuide.initializedStateWithGuide.NAME) or 0
+  if initialized == initializedStateWithGuide.initializedStateWithGuide.initialized.NAME then
+    device:send_to_component(command.component, WindowCovering.server.commands.GoToLiftPercentage(device, 0))
+  end
+end
+
+local function set_rotate_command_handler(driver, device, command)
+  device:emit_event(shadeRotateState.rotateState.idle()) -- update UI

```

Comment:
```suggestion
  device:emit_event(shadeRotateState.rotateState.idle({state_change = true})) -- update UI
```
Hi @sbkimaqara , Could you add `state_change = true` option to avoid network error on latest plugin?